storage/innobase/log/log0recv.cc

```diff
@@ -1751,13 +1773,26 @@ dberr_t recv_sys_t::find_checkpoint()
         continue;
       }
 
+      DBUG_EXECUTE_IF("use_old_checkpoint",
+		      if (log_sys.next_checkpoint_lsn > 0) {
+		        if (checkpoint_lsn > log_sys.next_checkpoint_lsn)
+			  goto skip_lsn;
+			else goto assign_lsn;
+		       }
+		     );
```

Comment:
This feels a little unsafe. What if there is only one usable checkpoint header in the file?